src/executor/stack/executor.rs

```diff
@@ -648,7 +648,7 @@ impl<'config, 'precompiles, S: StackState<'config>, P: PrecompileSet>
 		fn check_first_byte(config: &Config, code: &[u8]) -> Result<(), ExitError> {
 			if config.disallow_executable_format {
 				if let Some(0xef) = code.get(0) {
```

Comment:
```suggestion
				if let Some(0xef) = code.get(0) {
				if let Some(Opcode::INVALID.as_u8()) = code.get(0) {
```

Reply:
My bad, I've mistaken `INVALID` constant (`0xfe`) with the forbidden first byte (`0xef`) in contract creation. Just reverted that change.